socorro/cron/crontabber_app.py

```diff
@@ -46,17 +39,14 @@
 # Jobs that run in the -stage environment
 STAGE_JOBS = ', '.join(
     DEFAULT_JOBS_BASE + [
-        'socorro.cron.jobs.fetch_adi_alt.FAKEFetchADIFromHiveCronApp|1d|08:20',
         'socorro.cron.jobs.monitoring.DependencySecurityCheckCronApp|1d',
     ]
 )
 
 
 # Jobs that run in the -stage-new and -prod-new environments
 STAGE_NEW_JOBS = ', '.join(
-    DEFAULT_JOBS_BASE + [
-        'socorro.cron.jobs.fetch_adi_alt.RawADIMoverCronApp|1d|08:20'
-    ]
+    DEFAULT_JOBS_BASE + []
```

Comment:
Two things. First, you could skip the `+ []` part since that's not really doing anything.

Second, if this is `STAGE_NEW_JOBS`, I wonder what prod is running now and it's probably worth writing up a bug and making sure all the environments are running the right set of cron jobs.

Also, also, it's worth noting that if we ditch the dependency security check cron app, then all our environments are the same which would be nice. That doesn't affect this PR at all, but I figured I'd mention it so someone remembers later.